app/src/main/cpp/skyline/loader/nro.cpp

```diff
@@ -6,6 +6,7 @@
 #include <kernel/memory.h>
 #include <vfs/nacp.h>
 #include <vfs/region_backing.h>
+#include "executable.h"
```

Comment:
This include is redundant as `executable.h` is included in `loader.h` which is included here via `nro.h`